From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 18 Jul 2023 18:39:01 +0200
Subject: Use subprocess instead of subprocess32

This is a compatibility module for Python 2.7, which isn't a use-case we
are about in the tests in Debian. subprocess32 isn't available in
Debian.

Forwarded: not-needed
---
 tests/testlib.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/testlib.py b/tests/testlib.py
index 83629c3..5e9d0a7 100644
--- a/tests/testlib.py
+++ b/tests/testlib.py
@@ -18,7 +18,7 @@ except ImportError:
     import ConfigParser as configparser
 
 import psutil
-import subprocess32 as subprocess
+import subprocess
 
 import mitogen.core
 import mitogen.fork
